### PR TITLE
showImages: Allow iframe overflow

### PR DESCRIPTION
**Pull request by larsjohnsen**
Removes the width restriction imposed on it by `.entry`, similar to videos and images.

### CODE REVIEW

File: lib/modules/showImages.js [hunk 1]
```diff
@@ -1118,13 +1118,17 @@ function generateIframe(options) {
...
-	iframeNode.height = options.height || '360';
+	iframeNode.style.minWidth = options.width || '640px';
+	iframeNode.style.height = options.height || '360px';
 	iframeNode.allowFullscreen = true;
 	iframeNode.setAttribute('frameborder', '0');
-	iframeNode.style.maxWidth = options.maxwidth || '100%';
 
 	iframeNode.expand = () => {
+		iframeNode.dispatchEvent(new CustomEvent('resize', { bubbles: true }));
+		if (iframeNode.style.minWidth.match(/%$/) && iframeNode.clientWidth < 480) {
```
Comment (erikdesjardins) [inline lib/modules/showImages.js]: `iframeNode.style.minWidth.endsWith('%')`

Comment (erikdesjardins) [inline lib/modules/showImages.js]: Actually, could we accomplish the same thing by doing:
```js
iframeNode.style.minWidth = '480px';
iframeNode.style.width = options.width || '640px';
```
above?

Or is there a reason that `options.width` needs to be the `minWidth`?

Comment (larsjohnsen) [inline lib/modules/showImages.js]: `iframe` behaves weirdly (at least in Chrome) when parent has a width-limit. Fixed it by wrapping it in a non width limited `div` instead, in latest commit.

Comment (larsjohnsen) [inline lib/modules/showImages.js]: Never mind, that did not actually work. Reverted it, and added a comment instead.

**Pull request merged**